packages/lesswrong/components/walledGarden/WalledGardenPortalBar.tsx

```diff
@@ -90,18 +89,14 @@ export const WalledGardenPortalBar = ({iframeRef, classes}:{iframeRef:React.RefO
           <div><GardenCodeWidget type="event"/></div>
         </div>
       </div>}
-      {currentUser.walledGardenInvite && <div className={classes.eventWidget} onClick={() => refocusOnIframe()}>
-        <GardenCodesList terms={{view:"semipublicGardenCodes", types: ['public', 'semi-public']}} />
+      {currentUser.walledGardenInvite && <div className={classes.eventWidget}>
+        <GardenCodesList/>
+        <GardenCodesList personal/>
       </div>}
       <div className={classes.codesList}>
```

Comment:
This should very likely be deleted

Reply:
Yeah, that shouldn't be there.

Reply:
Wait, sorry. Not paying attention. What should be deleted?

The repeated component is correct.

Reply:
Hmm, Github preview is a bit silly. There is an empty div with the class `codesList` here, that has a child that you commented out in this PR. It seems kinda unlikely you still want this empty div.

Reply:
I'm not sure what that means. Don't have the context for what should be deleted or why

Reply:
Ah, yep. Vestigial stuff. Sorry.

Reply:
It was from when there were separate columns/lists for public events and an individual's events.

Reply:
Ruby are you owning fixing whatever this is?

Reply:
I already did.

Reply:
(it says "outdated" now)